app/components/request/configs.js

```diff
@@ -0,0 +1,141 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { equal, or, reads } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
+import truncate from 'travis/utils/computed';
+import CanTriggerBuild from 'travis/mixins/components/can-trigger-build';
+import TriggerBuild from 'travis/mixins/components/trigger-build';
+
+export const STATUSES = {
+  CLOSED: 'closed',
+  CUSTOMIZE: 'customize',
+  PREVIEW: 'preview'
+};
+
+export default Component.extend(CanTriggerBuild, TriggerBuild, {
+  classNames: ['request-configs'],
+  classNameBindings: ['status'],
+
+  preview: service('request-config'),
```

Comment:
yes, i've renamed things back and forth a couple times. i have settled on this naming because i felt it best to name the service to be named consistently across all related components ... and the service itself and the component `request-config-preview` _also_ have a property `requestConfig` for the model that is returned from `/repo/:slug/request/config` ... so there's some naming conflicts here. 

i think once it is understood that `preview` is the service then `preview.rawConfigs`, `preview.requestConfig` etc reads quite well though, so i think this is an ok compromise.